docs/manual-installation.md

```diff
@@ -0,0 +1,123 @@
+
+On this tutorial we'll see how to install rbenv, PostgreSQL and Decidim, and how to configure everything together.
+
+### Installing rbenv
+
+First we're going to install rbenv, for managing various ruby versions. We're following the guide from [DigitalOcean](https://www.digitalocean.com/community/tutorials/how-to-install-ruby-on-rails-with-rbenv-on-ubuntu-16-04).
+
+```bash
+sudo apt-get update
+sudo apt-get install -y build-essential autoconf bison build-essential libssl-dev libyaml-dev libreadline6-dev zlib1g-dev libncurses5-dev libffi-dev libgdbm3 libgdbm-dev
+git clone https://github.com/rbenv/rbenv.git ~/.rbenv
+echo 'export PATH="$HOME/.rbenv/bin:$PATH"' >> ~/.bashrc
+echo 'eval "$(rbenv init -)"' >> ~/.bashrc
+source ~/.bashrc
+git clone https://github.com/rbenv/ruby-build.git ~/.rbenv/plugins/ruby-build
+rbenv install 2.5.0
+rbenv global 2.5.0
+echo "gem: --no-document" > ~/.gemrc
+gem install bundler
+```
+
+### Installing Decidim
+
+Next, we need to install the `decidim` gem:
+
+```bash
+gem install decidim
+```
+
+Afterwards, we can create an application with the nice `decidim` executable, where `decidim_application` is your application name (ie DecidimBarcelona):
+
+```bash
+decidim decidim_application
+cd decidim_application
+bundle install
+```
+
+We recommend that you save it all on Git.
+
+```bash
+git init .
+git commit -m "Initial commit. Generated with Decidim 0.X https://decidim.org"
+```
+
+### Installing PostgreSQL
+
+Now we're going to install PostgreSQL for the database:
+
+```bash
+sudo apt-get install -y postgresql libpq-dev
+sudo su - postgres
+psql
+> CREATE USER decidim_app WITH CREATEROLE SUPERUSER CREATEDB;
+> \password decidim_app
+```
+
+You need to enter a password (two times) and save it somewhere to configure it later with the application.
+
+### Configure the database
+
+You need to modify your secrets (see `config/database.yml`). For this you can use [figaro](), [dotenv](https://github.com/bkeepers/dotenv) or [rbenv-vars](https://github.com/rbenv/rbenv-vars). You should always be careful of not uploading your secrets on git or your version control system.
+
+For instance, for working with figaro, add this to your `Gemfile`:
+
+```
+gem "figaro"
+```
+
+Then install it:
+
+```bash
+bundle install
+bundle exec figaro install
+```
+
+Next add this to your `config/application.yml`, using the setup the PostgreSQL database name, user and password that you configure before.
+
+```
+DATABASE_HOST: localhost
+DATABASE_USERNAME: decidim_app
+DATABASE_PASSWORD: your_password
+```
+
+Finally, save it all to git:
+
+```bash
+git add .
+git commit -m "Adds figaro configuration management"
+```
+
+### Initializing your app for local development
```

Comment:
Maybe here we should link to this space on getting started, so we don't duplicate the text, but the idea here was to be able to explain more what and where exactly you can log in and what can do every user.